From bbfc36f06c191d3bdea19cacc498afb86bd25404 Mon Sep 17 00:00:00 2001
From: Craig Andrews <candrews@integralblue.com>
Date: Thu, 5 Dec 2019 14:49:50 -0500
Subject: [PATCH] Add missing cassert includes
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Without these includes, ‘assert’ was not declared in this scope errors
occur.
---
 xbmc/profiles/dialogs/GUIDialogProfileSettings.cpp | 1 +
 xbmc/windowing/GraphicContext.cpp                  | 2 ++
 2 files changed, 3 insertions(+)


--- xbmc-18.4-Leia/xbmc/profiles/dialogs/GUIDialogProfileSettings.cpp~	2019-08-31 11:34:13.000000000 +0000
+++ xbmc-18.4-Leia/xbmc/profiles/dialogs/GUIDialogProfileSettings.cpp	2021-08-20 09:13:44.012432476 +0000
@@ -8,6 +8,7 @@

 #include "GUIDialogProfileSettings.h"
 
+#include <cassert>
 #include <utility>
 
 #include "dialogs/GUIDialogFileBrowser.h"
--- xbmc-18.4-Leia/xbmc/windowing/GraphicContext.cpp~	2019-08-31 11:34:13.000000000 +0000
+++ xbmc-18.4-Leia/xbmc/windowing/GraphicContext.cpp	2021-08-20 09:16:14.469325393 +0000
@@ -24,6 +24,8 @@
 #include "guilib/GUIWindowManager.h"
 #include "guilib/TextureManager.h"
 
+#include <cassert>
+
 using namespace KODI::MESSAGING;
 
 CGraphicContext::CGraphicContext(void) = default;
